doc/sphinx/proofs/writing-proofs/reasoning-inductives.rst

```diff
@@ -494,6 +500,11 @@ and an explanation of the underlying technique.
 
 .. seealso:: :tacn:`functional induction`
 
+.. _equality-inductive_types:
+
+:tacn:`discriminate` and :tacn:`injection`
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
```

Comment:
If you don't mind, let's also keep this new subsection title for the next PR (because for now it doesn't correctly reflect the content below---given that `inversion`, `fix` and `cofix` are also there). Since you need the label, I suggest moving it down above the "Equality and inductive sets" section title.